src/settings.js

```diff
@@ -131,6 +131,7 @@ var ABORTING_MALLOC = 1;
 // If 1, generated a version of memcpy() and memset() that unroll their
 // copy sizes. If 0, optimizes for size instead to generate a smaller memcpy.
 // This flag only has effect when targeting asm.js.
+// This setting is only meaningful with the fastcomp backend.
```

Comment:
How about we use the same style as `[compile+link]`.   E.g. `[fastomp-only]` and `[upstream-only]`?

That we could, as a followup, automate the error messages.

Reply:
Done.

(I don't think we need "upstream only" as soon there will only be upstream, and people should be using it already anyhow.)